integration/tests/common/mvp_test.go

```diff
@@ -126,9 +129,53 @@ func runMVPTest(t *testing.T, ctx context.Context, net *testnet.FlowNetwork) {
 			newAccountAddress = accountCreatedEvent.Address()
 		}
 	}
-
+	
 	fmt.Printf("new account address: %s\n", newAccountAddress)
 
+	// Generate the fund account transaction (so account can be used as a payer)
+	fundAccountTx := sdk.NewTransaction().
+		SetScript([]byte(fmt.Sprintf(`
+			import FungibleToken from 0x%s
+			import FlowToken from 0x%s
+
+			transaction(amount: UFix64, recipient: Address) {
+			  let sentVault: @FungibleToken.Vault
+			  prepare(signer: AuthAccount) {
+				let vaultRef = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)
+				  ?? panic("failed to borrow reference to sender vault")
+				self.sentVault <- vaultRef.withdraw(amount: amount)
+			  }
+			  execute {
+				let receiverRef =  getAccount(recipient)
+				  .getCapability(/public/flowTokenReceiver)
+				  .borrow<&{FungibleToken.Receiver}>()
+					?? panic("failed to borrow reference to recipient vault")
+				receiverRef.deposit(from: <-self.sentVault)
+			  }
+			}`,
+			fvm.FungibleTokenAddress(chain).Hex(),
+			fvm.FlowTokenAddress(chain).Hex()))).
+		AddAuthorizer(serviceAddress).
+		SetReferenceBlockID(sdk.Identifier(root.ID())).
+		SetProposalKey(serviceAddress, 0, serviceAccountClient.GetSeqNumber()).
+		SetPayer(serviceAddress)
+
+	err = fundAccountTx.AddArgument(cadence.UFix64(1_0000_0000))
```

Comment:
more of question than a comment, but why 1_0000_0000 instead of 100_000_000?

Reply:
I guess to indicate 1.00000000

Reply:
Yep. I find it easier to read flow values with a separator after every four zeros. that makes 1_0000_0000 exactly 1 flow and then I can go from there.